Source/WebKit/UIProcess/API/gtk/WebKitWebViewBase.cpp

```diff
@@ -1350,12 +1389,13 @@ static gboolean handleScroll(WebKitWebViewBase* webViewBase, double deltaX, doub
         // We only have hold events on touchpads, so assume one if we get no event.
         GdkInputSource source = device ? gdk_device_get_source(device) : GDK_SOURCE_TOUCHPAD;
 
+        bool isEnd = event ? gdk_event_is_scroll_stop_event(event) : false;
```

Comment:
And since null events only happen for scroll-end, `false` is absolutely wrong here.

Reply:
Changed to true as the fallback then.